includes/class-wc-stripe-webhook-handler.php

```
@@ -1375,8 +1382,18 @@ protected function handle_deferred_payment_intent_succeeded( $order, $intent_id
 			'The wc_gateway_stripe_process_payment action is deprecated. Use wc_gateway_stripe_process_payment_charge instead.'
 		);
 
+		$should_update_status = false;
```

PHPMD (2.15.0)

[error] The variable $should_update_status is not named in camelCase. (undefined)

(CamelCaseVariableName)